README.rst

```diff
@@ -1,84 +1,138 @@
+   * Option 2: In AWS, create a Lambda function, then attach a SignalFx-hosted layer with a wrapper.
+   * Option 3: In AWS, create a Lambda function, then create and attach a layer based on a SignalFx SAM (Serverless Application Model) template.
+
+Review the appropriate option below.
+
+**Option 1: Install the wrapper**
+
+Run the following installation script in your command line:
 
 .. code::
 
     pip install signalfx_lambda
 
-Configuring the ingest endpoint
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~
 
-By default, this function wrapper will send to the ``us0`` realm. If you are
-not in this realm you will need to set the ``SIGNALFX_INGEST_ENDPOINT`` environment
-variable to the correct realm ingest endpoint (``https://ingest.{REALM}.signalfx.com``).
-To determine what realm you are in, check your profile page in the SignalFx
-web application (click the avatar in the upper right and click My Profile).
+**Option 2: Create a Lambda function, then attach the SignalFx-hosted layer**
 
+In this option, you will use a layer created and hosted by SignalFx.
 
-Environment Variables
-~~~~~~~~~~~~~~~~~~~~~
+1. To verify compatibility, review the list of supported regions. See [Lambda Layer Versions](https://github.com/signalfx/lambda-layer-versions/blob/master/python/PYTHON.md).
+2. Open your AWS console. 
+3. In the landing page, under **Compute**, click **Lambda**.
+4. Click **Create function** to create a layer with SignalFx's capabilities.
+5. Click **Author from scratch**.
+6. In **Function name**, enter a descriptive name for the wrapper. 
+7. In **Runtime**, select the desired language.
+8. Click **Create function**. 
+9. Click on **Layers**, then add a layer.
+10. Mark **Provide a layer version**.
+11. Enter an ARN number. 
+  * To locate the ARN number, see [Lambda Layer Versions](https://github.com/signalfx/lambda-layer-versions/blob/master/python/PYTHON.md).
 
-**Note: the environment variables ``SIGNALFX_INGEST_ENDPOINT`` and ``SIGNALFX_AUTH_TOKEN`` are being deprecated and will not be supported in future releases.**
+~~~~~
 
-.. code:: bash
+**Option 3: Create a Lambda function, then create and attach a layer based on a SignalFx template**
 
-    SIGNALFX_ACCESS_TOKEN=access token
+In this option, you will chose a SignalFx template, and then deploy a copy of the layer.
+
+1. Open your AWS console. 
+2. In the landing page, under **Compute**, click **Lambda**.
+3. Click **Create function** to create a layer with SignalFx's capabilities.
+4. Click **Browse serverless app repository**.
+5. Click **Public applications**.
+6. In the search field, enter and select **signalfx-lambda-python-wrapper**.
+7. Review the template, permissions, licenses, and then click **Deploy**.
+    * A copy of the layer will now be deployed into your account.
+8. Return to the previous screen to add a layer to the function, select from list of runtime compatible layers, and then select the name of the copy.  
+
+
+~~~~~
+
+Step 2: Update the SignalFx realm
```

Comment:
Apologies, I unintentionally I misled you - I've been double checking various setup options and my proposed changes were not that good. 
So: the realm only matters if a user wishes to send metric directly to ingest endpoint (as opposed to using their gateway or pops). So maybe it would be better to go back on that and say sth in the lines of:

Setp 2: Locate correct ingest endpoint. 
By default, this function wrapper will send data to the ``us0`` realm. If you are not in this realm, and wish to use ingest endpoint directly, please make sure to point to the correct realm by using a subdomain, such as us1.signalfx.com ( {REALM}.signalfx.com ) - for example, ingest.us1.signalfx.com or ingest.eu0.signalfx.com.

Reply:
Thank you for the clarification! This paragraph has been updated.